### DISCUSSION TITLE
Why are box shifted when cropped?

### DISCUSSION THREAD

**Discussion by BenoitdeKersabiec** [mindee/doctr#1172] [General]
The forward function in `OCRPredictor` uses (if `assume_straight_pages`) the function `extract_crops` defined [here](https://github.com/mindee/doctr/blob/3a270b0cc4f4294f687fe63ae4c5e11dcdc8edab/doctr/utils/geometry.py#L315) to extract the crops of the detected words from the original image.

In this function, and in the if statement, we first denormalize the bounding boxes and round them to integers. 
However, I don't understand [line 338](https://github.com/mindee/doctr/blob/3a270b0cc4f4294f687fe63ae4c5e11dcdc8edab/doctr/utils/geometry.py#L338) that does `_boxes[2:] += 1` with the comment `"Add last index"`, it looks to me like we are arbitrarily shifting all the boxes but the first two ones. 

What is the purpose of this shift? is it wanted? I understand that one pixel isn't a lot, but it's a weird behavior

**Comment by felixdittrich92** [comment]
ping @charlesmindee

**Comment by BenoitdeKersabiec** [comment]
Any update here? Should I create an issue about this?
